src/main/java/com/minecolonies/coremod/colony/buildings/BuildingBuilder.java

```diff
@@ -300,12 +301,12 @@ private void updateAvailableResources()
             //Count in the additional chests as well
             if (builder!=null)
             {
-                for(final BlockPos pos : getAdditionalCountainers())
+                for(final BlockPos pos : getAdditionalContainers())
                 {
                     final TileEntity entity = builder.worldObj.getTileEntity(pos);
                     if(entity instanceof TileEntityChest)
                     {
-                        resource.addAvailable(InventoryUtils.getItemCountInInventory((TileEntityChest)entity, resource.getItem(), resource.getDamageValue()));
+                        resource.addAvailable(InventoryUtils.getItemCountInInventory(entity.getCapability(CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, null), resource.getItem(), resource.getDamageValue()));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Split this 208 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)